src/services/routes/org.routes.spec.ts

```diff
@@ -330,4 +330,77 @@ describe('org routes', () => {
       expect(response).toHaveProperty('statusCode', HTTP_STATUS_CODES.NO_CONTENT);
     });
   });
+
+  describe('get/:orgName', () => {
+    const injectionOptions: InjectOptions = {
+      method: 'GET',
+    };
+
+    test.each([
+      ['ASCII', ORG_NAME],
+      ['Non ASCII', NON_ASCII_ORG_NAME],
+    ])('%s name should return an org', async (_type, name: string) => {
+      const getOrgSuccessResponse = {
+        didSucceed: true,
+
+        result: {
+          name,
+          memberAccessType: 'INVITE_ONLY',
+        },
+      } as const;
```

Comment:
If we use `as const` here, we're creating a new type, which we don't want to do. Instead, we should explicitly set the type of this variable (`SuccessfulResult<Org...>`).